SportsResults.frockett/SportsResults.frockett/EmailComposer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SportsResults.frockett.Models;
+using System.Text;
+
+namespace SportsResults.frockett;
+
+internal static class EmailComposer
+{
+    public static string BuildHtmlTable(List<GameResult> results)
+    {
+        var htmlBuilder = new StringBuilder();
```

Comment:
Good use of string builder pattern I must say :smile:

Reply:
Sorry does this mean you think I should change something or is it just a compliment? haha

Reply:
it's a compliment lmao :grin: